Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1069,6 +1070,8 @@ void TextureCacheBase::CopyRenderTargetToTexture(u32 dstAddr, unsigned int dstFo
 	bool copy_to_ram = !g_ActiveConfig.bSkipEFBCopyToRam;
 	bool copy_to_vram = true;
 
+	g_renderer->GetPostProcessor()->OnEFBCopy();
+
```

Comment:
A good euristic to correctly apply post-processing here would be 
srcFormat != PEControl::PixelFormat::Z24 && !isIntensity && (dstFormat == 4 || dstFormat == 5 || dstFormat == 6)